### PR TITLE
[CLEANUP] Always escape backslashes in strings

### PR DESCRIPTION
**Pull request by oliverklee**
This was done using the `string_implicit_backslashes` PHP-CS-Fixer rule.

### CODE REVIEW

File: src/Property/KeyframeSelector.php [hunk 1]
```diff
@@ -14,12 +14,12 @@ class KeyframeSelector extends Selector
...
-            [a-zA-Z0-9\x{00A0}-\x{FFFF}_^$|*="\'~\[\]()\-\s\.:#+>]* # any sequence of valid unescaped characters
-            (?:\\\\.)?                                              # a single escaped character
-            (?:([\'"]).*?(?<!\\\\)\2)?                              # a quoted text like [id="example"]
+            [a-zA-Z0-9\\x{00A0}-\\x{FFFF}_^$|*="\'~\\[\\]()\\-\\s\\.:#+>]* # any sequence of valid unescaped characters
+            (?:\\\\.)?                                                     # a single escaped character
+            (?:([\'"]).*?(?<!\\\\)\\2)?                                    # a quoted text like [id="example"]
         )*
     )|
-    (\d+%)                                                          # keyframe animation progress percentage (e.g. 50%)
+    (\\d+%)                                                          # keyframe animation progress percentage (e.g. 50%)
```
Comment (JakeQZ) [inline src/Property/KeyframeSelector.php:22]: I think the comment here neeeds moving to align with the others.

File: src/Property/Selector.php [hunk 2]
```diff
@@ -14,11 +14,11 @@ class Selector
...
      */
     private const NON_ID_ATTRIBUTES_AND_PSEUDO_CLASSES_RX = '/
-        (\.[\w]+)                   # classes
+        (\\.[\\w]+)                   # classes
         |
-        \[(\w+)                     # attributes
+        \\[(\\w+)                     # attributes
         |
-        (\:(                        # pseudo classes
+        (\\:(                        # pseudo classes
```
Comment (JakeQZ) [inline src/Property/Selector.php:21]: Same here.

File: src/Property/Selector.php [hunk 3]
```diff
@@ -38,9 +38,9 @@ class Selector
      * @var string
      */
     private const ELEMENTS_AND_PSEUDO_ELEMENTS_RX = '/
-        ((^|[\s\+\>\~]+)[\w]+   # elements
+        ((^|[\\s\\+\\>\\~]+)[\\w]+   # elements
         |
-        \:{1,2}(                # pseudo-elements
+        \\:{1,2}(                # pseudo-elements
```
Comment (JakeQZ) [inline src/Property/Selector.php:43]: And here.

**Review by JakeQZ — CHANGES_REQUESTED** [review]
In some of the regex patterns with the `PCRE_EXTENDED` modifier, the comments have become misaligned.

**Comment by oliverklee** [comment]
Fixed the alignments and repushed.

**Review by JakeQZ — APPROVED** [review]
Looks fine now.

**Pull request merged**